addons/payment/migrations/12.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,31 @@
+---Fields in module 'payment'---
+payment      / account.invoice          / transaction_ids (many2many)   : NEW relation: payment.transaction
+payment      / payment.acquirer         / qr_code (boolean)             : NEW 
+# NOTHING TO DO
+
+payment      / payment.transaction      / date (datetime)               : NEW
+payment      / payment.transaction      / date_validate (datetime)      : DEL
+# DONE: pre-migration: renamed field
+
+payment      / payment.transaction      / invoice_ids (many2many)       : NEW relation: account.invoice
+payment      / payment.transaction      / is_processed (boolean)        : NEW 
```

Comment:
We should assume all the current transactions are processed, so please set this value by SQL

Reply:
I am not sure, but if you says so...